apps/angular/src/app/features/anime/anime-page/anime-page.component.html

```diff
@@ -0,0 +1,46 @@
+<camp-spinner *ngIf="isLoading$ | async"></camp-spinner>
```

Comment:
We can leverage self-closing tags
```suggestion
<camp-spinner *ngIf="isLoading$ | async" />
```